### PR TITLE
Create LICENSE file

### PR DESCRIPTION
**Pull request by DonatJR**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 SailReal / DonatJR
```
Comment (SailReal) [inline LICENSE]: Should we not put our real names her inside? 
Maybe we will change our github account names sometimes in the  future?

Comment (DonatJR) [reply LICENSE]: I created this using the github license template which uses the username automatically, but we could of course use our real names - do you want me to change it?

File: LICENSE [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 SailReal / DonatJR
+Copyright (c) 2018 Jonas Reinwald <jonas.reinwald@htwg-konstanz.de> / Julian Raufelder <julian.raufelder@htwg-konstanz.de>
```
Comment (SailReal) [inline LICENSE]: Does that fit to you?

Comment (DonatJR) [reply LICENSE]: looks fine to me :)

**Pull request merged**